agent/util-scripts/pbench-postprocess-tools

```diff
@@ -104,8 +107,11 @@ while read hostentry ;do
 		tools[${atool[0]}]=${atool[@]:1}
 	done
 
-	# FIXME: add support for label applied to the hostname directory.
-	host_tool_output_dir="${tool_output_dir}/${host}"
+	if [[ -z "${label}" ]] ;then
+		host_tool_output_dir="${tool_output_dir}/${host}"
+	else
+		host_tool_output_dir="${tool_output_dir}/${label}:${host}"
+	fi
```

Comment:
I'm sorry - I can't remember these things, so every time I come to one, my brain comes to a screeching  halt. I realize it's shorter, but every time I see one I have to consult the bash man page to remember what it does (and after 5 minutes the memory is gone again).

Reply:
That's fine.  It was just "an alternate possibility".  It _is_ more arcane, which means that although the brevity is attractive it might be "not better".